### PR TITLE
Product Terms Update

### PR DESCRIPTION
**Pull request by KasraKhosravi**
Hey

I just did some changes to current terms in product section and added few other terms.

### CODE REVIEW

File: product.md [hunk 1]
```diff
@@ -2,8 +2,23 @@ Product Terms
 =======================
 **[Back to index](README.md)**
 
-* Attributes: ویژگی ها
-* Options: آپشن ها
+* Product: محصول
+
+* Attributes: مشخصات
+* Options: انتخاب ها - آپشن ها
```
Comment (aramalipoor) [inline product.md]: For this one maybe we can go with قابلیت ها ! Or maybe خصیصه!
I don't think انتخاب ها is a good term! 

We can have different variations by mixing a product's Options so:
E.g.
از بین انتخاب های زیر چند انتخاب را برای محصول خود انتخاب کنید!
از بین خصیصه های زیر چند خصیصه را برای محصول خود انتخاب کنید!

با ترکیب انتخاب های یک محصول میتوانید ترکیبات مختلف بسازید!
با ترکیب خصیصه های یک محصول میتوانید ترکیبات مختلف بسازید!

BTW ترکیبات is under question itself ;) But that's another discussion.
@soheilc @ahmadrabie

**Comment by KasraKhosravi** [comment]
Good thinking. What I had in mind was "امکانات" but I didn't change it in the md file.

For instance we say this car has these options. which means this car has these powers and enablers.

"قابلیت ها" is also good. "خصیصه" is fuzzy and confusing in my mind.

Lets get back to "ترکیبات" later ;)

**Comment by ahmadrabie** [comment]
Attributes: خصوصیات

**Comment by soheilc** [comment]
@aramalipoor , What do you want to introduce by "options" word? in some cases it means setting but in other cases it means attributes. I think, translation of this word it's depends on where you use options word.

**Pull request merged**